### PR TITLE
add MoleculeFormats example SO--

### PR DESCRIPTION
**Pull request by mhellstr**

### CODE REVIEW

File: doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+
+Load PLAMS Molecule from .xyz file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: ipython3
+
+    from scm.plams import Molecule, plot_molecule
+    
+    mol = Molecule(xyz_file)
+    print(type(mol))
```
Comment (dormrod) [inline doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst]: `print(f"{type(mol)=}")`

File: doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+
+
+.. parsed-literal::
+
+    <class 'scm.plams.mol.molecule.Molecule'>
+
+
+.. parsed-literal::
+
+    /home/hellstrom/adfhome/bin/python3.8/lib/python3.8/site-packages/ase/io/cif.py:401: UserWarning: crystal system 'orthorhombic' is not interpreted for space group Spacegroup(74, setting=1). This may result in wrong setting!
```
Comment (dormrod) [inline doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst]: Is this warning expected and should it be included in the notebook output?

File: doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst [hunk 3]
```diff
@@ -0,0 +1,665 @@
...
+.. parsed-literal::
+
+    data_image0
+    _chemical_formula_structural       O16Si8
+    _chemical_formula_sum              "O16 Si8"
+    _cell_length_a       9.873
+    
+
+
+AMS .in system blockat format
```
Comment (dormrod) [inline doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst]: "system block format"?

File: doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst [hunk 4]
```diff
@@ -0,0 +1,665 @@
...
+.. parsed-literal::
+
+    type(mol)=<class 'scm.plams.mol.molecule.Molecule'>
+
+
+
+.. image:: MoleculeFormats_files/MoleculeFormats_31_1.png
+
+
+RDKIt Mol Python class
```
Comment (dormrod) [inline doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst]: typo -> RDKit

File: doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst [hunk 5]
```diff
@@ -0,0 +1,665 @@
...
+     19  Si    0.1572  0.75  0.9007
+     20  Si    0.6572  0.75  0.5993
+     21  Si    0.1572  0.25  0.0993
+     22  Si    0.3428  0.75  0.5993
+     23  Si    0.8428  0.25  0.0993
+
+
+.. parsed-literal::
+
+    /home/hellstrom/adfhome/bin/python3.8/lib/python3.8/site-packages/ase/io/cif.py:401: UserWarning: crystal system 'orthorhombic' is not interpreted for space group Spacegroup(74, setting=1). This may result in wrong setting!
```
Comment (dormrod) [inline doc/source/examples/MoleculeFormats/MoleculeFormats.ipynb.rst]: Same comment on the warning here

**Review by dormrod — CHANGES_REQUESTED** [review]
Looks good. Do you think this is better here than say in the cookbook, where there is already one similar example for the SMILES conversion: https://www.scm.com/doc/plams/cookbook/cookbook.html#generate-a-molecule-from-a-smiles-string?

If so, is that cookbook still useful, or should some of the snippets here be linked/included there?

**Comment by mhellstr** [comment]
Thanks, I fixed the small issues. The warning is still there, but it is generated by ASE, so I guess it's up to the user to figure out if it's important. (Doesn't seem important to me)

In principle I think the cookbook should just be made into one or more notebook examples.

**Review by dormrod — APPROVED** [review]

**Pull request merged**